app/routes.py

```diff
@@ -0,0 +1,345 @@
+            "name" : customer.name,
+            "phone" : customer.phone,
+            "postal_code" : customer.postal_code
+        })
+    return jsonify(video_rentals_response)
+
+
+@videos_bp.route("", methods=["GET", "POST"])
+def handle_videos():
+    if request.method == "GET":
+        videos = Video.query.all()
+        videos_response = []
+        for video in videos:
+            videos_response.append({
+                "id": video.id,
+                "title": video.title,
+                "release_date": video.release_date,
+                "total_inventory": video.total_inventory
+            })
+        
+        return jsonify(videos_response), 200
+
+    elif request.method == "POST":
+        request_body = request.get_json()
+        title = request_body.get("title")
+        release_date = request_body.get("release_date")
+        total_inventory = request_body.get("total_inventory")
+
+        if not title:
+            return jsonify({
+                "details" : "Request body must include title."
+            }), 400
+        
+        if not release_date:
+            return jsonify({
+                "details" : "Request body must include release_date."
+            }), 400
+        
+        if not total_inventory:
+            return jsonify({
+                "details" : "Request body must include total_inventory."
+            }), 400
+        
+        new_video = Video(
+            title=title,
+            release_date=release_date,
+            total_inventory=total_inventory
+        )
+        db.session.add(new_video)
+        db.session.commit()
+
+        return {
+            "id": new_video.id,
+            "title" : new_video.title,
+            "total_inventory" : new_video.total_inventory
+        }, 201
+
+
+@videos_bp.route("/<video_id>", methods= ["GET", "PUT", "DELETE"])
+def handle_video(video_id):
+    if not video_id.isnumeric():
+        return jsonify(None), 400
+
+    video = Video.query.get(video_id)
+
+    if video is None:
+        return jsonify({
+            "message" : f"Video {video_id} was not found"
+        }), 404
+
+    if request.method == "GET":
+        return jsonify({
+            "id" : video.id,
+            "title" : video.title,
+            "release_date" : video.release_date,
+            "total_inventory" : video.total_inventory
+        }), 200
+    
+    elif request.method == "DELETE":
+        db.session.delete(video)
+        db.session.commit()
+
+        return jsonify({
+            "id" : video.id
+        }), 200
+
+    elif request.method == "PUT":
+        request_body = request.get_json()
+        title = request_body.get("title")
+        release_date = request_body.get("release_date")
+        total_inventory = request_body.get("total_inventory")
+
+        if not (title and release_date and total_inventory):
+            return jsonify({
+                "details" : "Invalid data"
+            }), 400
```

Comment:
👍 

Good job performing similar validations as for creating a video record through POST.